selfdrive/car/hyundai/carstate.py

```diff
@@ -143,6 +143,8 @@ def update(self, cp, cp_cam):
     # save the entire LKAS11 and CLU11
     self.lkas11 = copy.copy(cp_cam.vl["LKAS11"])
     self.clu11 = copy.copy(cp.vl["CLU11"])
+    self.fca11 = copy.copy(cp_cruise.vl["FCA11"])
+    self.fca12 = copy.copy(cp_cruise.vl["FCA12"])
```

Comment:
Can we not just forward these messages through panda without any modification?

Reply:
I believe we can. We only modify those messages to show the amber FCW icon on the dashboard at the moment. Since this does not disable radar, we probably could just forward these messages. @adeebshihadeh thoughts?

Reply:
until we confirm that AEB is unaffected, let's just treat them the same as a disabled radar, i.e. block in fwd hook and send our own (no copy)